site/content/en/docs/Installation/_index.md

```diff
@@ -79,11 +79,83 @@ You should be able to see the output below from your console. The `01-open-match
   - Several Services for Open Match endpoints. 
   - HorizontalAutoScalars to auto-scale Open Match based on pods' average CPU utilization. 
 
+## Install ConfigMap `om-configmap-override`
```

Comment:
This section has two stylistic pieces I don't like:
- Lots of printouts from running commands.
- Awkward prose for a technical document.  This is overly verbose, and follows a train of thought debugging a problem, instead of just explaining what the user should do next.

Suggested contents:
```
## Install ConfigMap `om-configmap-override`

Open Match requires configuration before all components will start.  If you are proceeding to getting started, or following a tutorial, it will contain the commands to apply the required configuration.

For writing your own, a yaml which is useful as a starting point for further edits is available at [06-open-match-override-configmap.yaml](https://open-match.dev/install/v{{< param release_version >}}/yaml/06-open-match-override-configmap.yaml).  To configure Open Match outside of a tutorial, download this file, prepare your edits, and run:
`kubectl apply -f <filename here>`

```